Suggested npm installation command failed

This command, suggested from [README.md] Installation section fails:
```
sudo npm install -g substance-io
```

#### Result
```
npm ERR! 404 Not Found
npm ERR! 404 
npm ERR! 404 'substance-io' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, or http url, or git url.

npm ERR! System Linux foo-bar-generic
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "substance-io"
npm ERR! cwd /home/foobar/substance.io/io
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.4.21
npm ERR! code E404
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/foobar/substance.io/io/npm-debug.log
npm ERR! not ok code 0
```

Either a problem of [npm] or [README.md] file.

[README.md]: https://github.com/substance/io/blob/b9e737687996a5680b7c90cfd68bf8fa43255497/README.md
[npm]: https://www.npmjs.com/

--- a/README.md
+++ b/README.md
@@ -1,15 +1,24 @@
-This project is deprecated.
+**This project is deprecated**.
+
+[substance]: https://github.com/substance/substance
 
 # Substance IO
 
 A minimal publishing solution based on Pandoc Markdown and Substance.
+
+## Development Status
+
+Development of [substance.io](http://substance.io)
+has been delegated to [another repository][substance].
+
+This repository is deprecated.
 
 ## Prerequisites
 
 - Node.js >=0.10.x
 - Pandoc >= 1.12.02 (for Markdown cross-compiling)
 
-## Install
+## Install (Deprecated)
 
 Substance.IO comes as a ready-to-use NPM module:
 

